packages/web3-providers/src/types/RSS3.ts

```diff
@@ -131,20 +138,83 @@ export namespace RSS3BaseAPI {
         id: string
         imageURL?: string
         title?: string
+        description?: string
+        actions?: Action[]
+        tokenAmount?: BigNumber
```

Comment:
Are you sure an API will return a bignumber value?

Reply:
tokenAmount is a param calculated after API response